protocol/messages/room_messages.go

```diff
@@ -0,0 +1,28 @@
+package messages
+
+import "github.com/jtieri/habbgo/protocol/packets"
+
+func INTERSTITIALDATA() *packets.OutgoingPacket {
+	p := packets.NewOutgoing(258) // Base64 Header DB
+
+	/*
+		This is in the client Lingo src:
+		on handle_interstitialdata me, tMsg
+		  if tMsg.content.length > 1 then
+		    tDelim = the itemDelimiter
+		    the itemDelimiter = "\t"
+		    tSourceURL = tMsg.content.getProp(#item, 1)
+		    tTargetURL = tMsg.content.getProp(#item, 2)
+		    the itemDelimiter = tDelim
+		    me.getComponent().getInterstitial().Init(tSourceURL, tTargetURL)
+		  else
+		    me.getComponent().getInterstitial().Init(0)
+		  end if
+		end
+
+		Interstitial may be used to do ads on loading screens while entering rooms?
```

Comment:
![pedobear](https://user-images.githubusercontent.com/981494/149616420-7015c088-721a-4df6-829b-43f5d4397933.png)
yes.

Reply:
HA! That's what I figured at a quick glance but looking at other v14 emulators it seemed like
A. Everyone else had these named incorrectly &
B. No one had implemented it correctly, if at all.